src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -192,49 +197,66 @@ def list_releases(name, resource_group_name):
     json_request = req.json()
     return json_request
 
+def _gitroot():
+    """
+    Gets the absolute path of the repository root
+    """
+    try:
+        base = check_output(['git', 'rev-parse', '--show-toplevel'])
+    except:
+        raise IOError('Current working directory is not a git repository')
+    return base.decode('utf-8').strip()
+
+def _get_filepath_in_current_git_repo(filename):
+    """
+    retrieves the full path of the first file in the git repo that matches filename
+    """
+    FILE_NAMES = 2
+    CWD = 0
+    for walk in os.walk(_gitroot()):
+        for file_name in walk[FILE_NAMES]:
+            if file_name.lower() == filename.lower():
```

Comment:
Suggest more descriptive name for the variable `filename`.
At a glance, `file_name.lower() == filename.lower()` looks like the same variables are being compared because the names are so similar.